### DISCUSSION TITLE
How to change the placeholder?

### DISCUSSION THREAD

**Discussion by SaraSanGar** [shishkin/astro-pagefind#82] [Q&A] [answered]
I have tried to change the placeholder in several ways but none of them work, could you help me? 

Thank you very much

**Comment by shishkin** [comment]
What ways did you try exactly?

This component allows passing options to Pagefind UI through `uiOptions` prop. You would need to put translations there as [documented by Pagefind](https://pagefind.app/docs/ui/#translations).

**Comment by SaraSanGar** [comment, marked as answer]
Thank you very much!! I was doing
```
<Search 
  className="online-classes__search-bar" 
  uiOptions={{ placeHolder: "What you want to learn?" }} 
/> 
```
Instead of 
```
<Search
  className="online-classes__search-bar"
  uiOptions={{
    translations: {
      placeholder: "What you want to learn?",
   },
 }}
/>
```
